index.html

```diff
@@ -187,8 +192,11 @@ <h2 class="main-heading">Images</h2>
           </div>
           <div class="center-heading">Round Image</div>
           <img class="round-image" src="Avatar images/handsome-patrik.jpg" />
-          <div class="center-heading-float">Above are the Images, which can be used in building of website.below is the html code I used to implement them</div>
-          <img class="html-image" src="code/image-html.png"/>
```

Comment:
The code would be more readable and manageable if it were divided into multiple files, by creating a separate file for each component.